1_fetch/cfg/wqp_codes.yml

```diff
@@ -0,0 +1,112 @@
+# characteristicNames: https://www.waterqualitydata.us/Codes/Characteristicname?mimeType=xml
+
+temperature:
+- "Temperature"
+- "Temperature, sample"
+- "Temperature, water"
+- "Temperature, water, deg F"
+
+phosphorus:
+- "Inorganic phosphorus"
+- "Phosphate"
+- "Phosphate-phosphorus"
+- "Phosphate-phosphorus as P"
+- "Phosphate-phosphorus as PO4"
+- "Phosphorus, phosphate (PO4) as orthophosphate"
+- "Soluble Reactive Phosphorus (SRP)"
+- "Polyphosphate"
+- "Phosphorus, hydrolyzable"
+- "Phosphorus, hydrolyzable plus orthophosphate"
+- "Organic phosphorus"
+- "Phosphorus, Particulate Organic"
+- "Phosphorus"
+- "Phosphorus as P"
+- "Total Phosphorus, mixed forms"
+- "Orthophosphate"
+- "Orthophosphate as PO4"
+- "Orthophosphate as P"
+- "Ortho-Phosphate-Phosphorus"
+
+nitrogen:
+- "Inorganic nitrogen (NO2, NO3, & NH3)"
+- "Inorganic nitrogen (ammonia, nitrate and nitrite)"
+- "Kjeldahl nitrogen"
+- "Nitrogen"
+- "Nitrogen Kjeldahl"
+- "Nitrogen ion"
+- "Nitrogen, ammonia (NH3) as NH4"
+- "Nitrogen, mixed forms (NH3), (NH4), organic, (NO2) and (NO3)"
+- "Nutrient-nitrogen"
+- "Organic Nitrogen"
+- "Particulate Organic Nitrogen and Particulate Nitrogen"
+- "Total Kjeldahl nitrogen"
+- "Total Kjeldahl nitrogen (Organic N & NH3)"
+- "Total Nitrogen, mixed forms (NH3), (NH4), organic, (NO2) and (NO3)"
+- "Total Particulate Nitrogen"
+- "Total Nitrogen, mixed forms"
+#- "Kjeldahl nitrogen as N"
+#- "Nitrogen as N"
+#- "Nitrogen, ammonia (NH3) + ammonium (NH4)"
+#- "Nitrogen, Ammonia + Organic"
+#- "Nitrogen, Ammonium (NH4) as N"
+#- "Nitrogen, ammonium (NH4) as NH4"
+#- "Nitrogen, Kjeldahl"
+#- "Nitrogen, mixed forms (NH3), (NH4), organic, (NO2) and (NO3) as N"
+#- "Nutrient-nitrogen as N"
+#- "Organic Nitrogen as N"
```

Comment:
Is there a specific reason why some of these are commented out?

Reply:
They're commented out because I borrowed this yml from `nawqa-wqp` and they were commented out [there](https://github.com/USGS-R/nawqa_wqp/blob/main/1_wqpdata/cfg/wqp_codes.yml) 😄. I haven't checked whether these commented out lines represent valid `characteristicNames` in WQP - @lindsayplatt, do you know? If they're valid names, I'll go ahead and uncomment them.

Reply:
I just tried searching for a couple in [this big list](https://www.waterqualitydata.us/Codes/Characteristicname?mimeType=xml) and don't see them there. Maybe we should just delete?

Reply:
Thanks for looking into that - I also tried a few basic `dataRetrieval` calls using a couple of the nitrogen ones and was getting errors back. I've opted to delete the commented-out `characteristicNames` from the yml file for now. If we find we're missing any, we can add new `characteristicNames` later.